web/styles/compose.css

```diff
@@ -213,8 +215,8 @@
 
 #compose_top {
     display: flex;
-    justify-content: space-between;
-    align-items: flex-start;
+    /* Preserve sensible tab order despite layout shift. */
+    flex-flow: row-reverse;
```

Comment:
How exactly does this work? I don't have strong feelings on what we want to do, but from an ease of understanding standpoint, the `order-1` bundle of classes was nice in that it was explicit and predictable, while still letting us freely arrange elements in the HTML source in a way that aligned with their appearance. 

(One idea that might be appropriate is to use an extra class with a name like `tab-order-reversed` that gets this property alongside `compose_top` just to better hint why it's this way in the source)

Reply:
I think an extra class is definitely a good idea, although I would call it something like `preserve-tab-order`, because that's what it's doing.

In other words, we're really _not_ free to put elements wherever we like in the HTML if we want to preserve a sensible tab order (I've got a similar fix that I need to put up for where the timestamp falls in the messagebox grid).

Although I didn't go digging in the commit history to prove it, my hunch is that that the old order attributes were put there to preserve the HTML structure that was needed when the layout was achieved through floats and positioning.

Reply:
Oh, and as to how it works, it just takes the normal source order in the HTML, where

A
B
C

gets laid out in a flexbox as ABC, and changes it instead to CBA. We're not doing anything more exotic than that (e.g., CAB), which would probably be a mistake anyway.